packages/fx-core/package-lock.json

```diff
@@ -1813,7 +1813,7 @@
     },
     "archy": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/archy/-/archy-1.0.0.tgz",
+      "resolved": "https://msazure.pkgs.visualstudio.com/9660fff2-2363-48b0-9e15-64df2283e932/_packaging/MODS-NPM-Registry/npm/registry/archy/-/archy-1.0.0.tgz",
```

Comment:
remove the npmrc file, seems the registry is not correct.

Reply:
fixed.